rust/crd/src/git_sync.rs

```diff
@@ -30,12 +30,7 @@ pub struct GitSync {
     pub git_sync_conf: Option<BTreeMap<String, String>>,
 }
 impl GitSync {
-    pub fn get_args(&self) -> Vec<String> {
-        let mut args: Vec<String> = vec![
-            COMMON_BASH_TRAP_FUNCTIONS.to_string(),
-            "prepare_signal_handlers".to_string(),
-        ];
-
+    pub fn get_args(&self, one_time: bool) -> Vec<String> {
```

Comment:
Please document what the `one_time` parameter is doing

Reply:
Added in c981177.